### DISCUSSION TITLE
Reduce the sidebar width

### DISCUSSION THREAD

**Discussion by Esarve** [Naezr/ShyFox#195] [Q&A]
Hi, I love this theme but the sidebar width is too much for my taste, I want to make it a bit tighter with reducing the pinned tab size as well. Is to possible to do this?

![image](https://github.com/user-attachments/assets/f6175232-fb58-4c37-8666-15dece233459)

**Comment by 0xda157** [comment]
The sidebar width can be changed very easily by editing this variable:
https://github.com/Naezr/ShyFox/blob/6488ff1934c184a7b81770c67f5c3b5e983152e3/chrome/ShyFox/shy-variables.css#L29

**Reply by mgrimace** [reply]
Not the OP but thanks for this, dropped mine to 200px (2/3) and it's perfect for me. @Esarve yes, it also scales the pin sizes down too
